dockerfiles/drbd-driver-loader/Dockerfile.bionic

```diff
@@ -1,7 +1,19 @@
 FROM ubuntu:bionic
-MAINTAINER Roland Kammerer <roland.kammerer@linbit.com>
 
-RUN apt-get update && apt-get upgrade -y && apt-get install -y kmod gnupg wget make gcc patch curl && apt-get clean
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install --no-install-recommends -y \
+      ca-certificates \
+      kmod \
+      gnupg \
+      wget \
+      make \
+      # Ubuntu has multiple kernel versions that may be using different gcc versions: use the dkms package to install them all
```

Comment:
that does works in docker files right, this does not become `make #I eat everything`. I'm sure you tested, it

Reply:
Yep, tested it, docker strips it before the shell sees anything.